.github/workflows/integration-test.yml

```diff
@@ -29,54 +29,57 @@ jobs:
       - uses: actions/checkout@v2
         with:
           path: toast
-      - uses: actions/checkout@v2
-        with:
-          repository: toastdotdev/www.toast.dev
-          path: www.toast.dev
       - name: Install nightly toolchain
         uses: actions-rs/toolchain@v1
         with:
           toolchain: nightly
           override: true
       - uses: actions/setup-node@v2-beta
         with:
-          node-version: '14'
+          node-version: "14"
           check-latest: true
       - name: Run cargo build
         working-directory: toast
-        run: cargo build --release
-      - name: tar the binary
-        working-directory: toast
+        run: cargo build
+      - name: Install fd
         run: |
-          tar -C target -cvzf ./target/${{matrix.platform}}.tar.gz release/${{matrix.target}}
-        shell: bash
-      - name: npm install in toast.dev
-        working-directory: www.toast.dev
-        run: npm ci
-      - name: copy and replace toast in node_modules
-        working-directory: www.toast.dev
-        run: cp -r ./../toast/toast-node-wrapper ./node_modules/toast
-        shell: bash
-      - name: postinstall on toast.dev
-        working-directory: www.toast.dev
-        run: npm run postinstall
-      - name: npm install in toast node wrapper
-        working-directory: toast
-        run: cd toast-node-wrapper && npm install
-      - name: set the binary path and install
-        working-directory: www.toast.dev
-        run: node ./../toast/toast-node-wrapper/binary-management/installDev.js ./../toast/target/${{matrix.platform}}.tar.gz
-      - name: toast incremental
-        working-directory: www.toast.dev
-        run: npm run build
-      - name: display files
-        working-directory: www.toast.dev/public
-        run: ls -R
+          cargo install fd-find
+      - name: starter
+        run: |
+          npx create-toast default
+          cd default
+          node -e 'const fs = require("fs"); const json = require("./package.json"); json.dependencies.toast = "file:../toast/toast-node-wrapper"; fs.writeFileSync("./package.json", JSON.stringify(json, null, 2));'
+          yarn
```

Comment:
Should we use npm here since we npx?

Reply:
It may be worth splitting these steps up as well so it's easier to see if the `create` fails, install fails (failed fetch is common enough) or in the build step.

Reply:
I was trying to use npm in the tests but the way they structured their node_modules with local file dependencies in CI was causing breadbox to fail. (`node_modules/toast/preact` was where preact was).

Will take a look at splitting these up. I want to expand this to other starters in the future.